### PR TITLE
Updated metrics test cases

### PR DESCRIPTION
**Pull request by cypo721**
Updated metrics test cases to new finder navigation.

### CODE REVIEW

File: basic/src/test/java/org/jboss/hal/testsuite/test/runtime/DataSourceMetricsTestCase.java [hunk 1]
```diff
@@ -30,19 +35,42 @@
...
+                    .addAddress(FinderNames.MONITOR, FinderNames.SUBSYSTEMS)
+                    .addAddress(FinderNames.SUBSYSTEM,"Datasources");
+        }
     }
 
     @Test
     public void connectionPoolMetrics(){
+        navigation.selectRow().invoke("View");
+        Application.waitUntilVisible();
+
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/runtime/DataSourceMetricsTestCase.java:71]: Please move this at the end of `before()` method.

File: basic/src/test/java/org/jboss/hal/testsuite/test/runtime/DataSourceMetricsTestCase.java [hunk 2]
```diff
@@ -56,6 +84,9 @@ public void connectionPoolMetrics(){
 
     @Test
     public void preparedStatementCacheMetrics(){
+        navigation.selectRow().invoke("View");
+        Application.waitUntilVisible();
+
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/runtime/DataSourceMetricsTestCase.java]: Please move this at the end of `before()` method.

File: basic/src/test/java/org/jboss/hal/testsuite/test/runtime/JvmMetricsTestCase.java [hunk 3]
```diff
@@ -29,19 +34,40 @@
...
+                    .addAddress(FinderNames.SERVER, FinderNames.STANDALONE_SERVER)
+                    .addAddress(FinderNames.MONITOR, "JVM");
+        }
     }
 
     @Test
     public void heapUsageMetrics(){
+        navigation.selectRow().invoke("View");
+        Application.waitUntilVisible();
+
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/runtime/JvmMetricsTestCase.java:68]: Please move this at the end of `before()` method.

File: basic/src/test/java/org/jboss/hal/testsuite/test/runtime/JvmMetricsTestCase.java [hunk 4]
```diff
@@ -54,6 +80,9 @@ public void heapUsageMetrics(){
 
     @Test
     public void threadUsageMetrics(){
+        navigation.selectRow().invoke("View");
+        Application.waitUntilVisible();
+
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/runtime/JvmMetricsTestCase.java]: Please move this at the end of `before()` method.

File: basic/src/test/java/org/jboss/hal/testsuite/test/runtime/TransactionsMetricsTestCase.java [hunk 5]
```diff
@@ -30,19 +35,42 @@
...
+                .addAddress(FinderNames.MONITOR, FinderNames.SUBSYSTEMS)
+                .addAddress(FinderNames.SUBSYSTEM,"Transactions");
+    }
     }
 
     @Test
     public void successRationMetrics() {
+        navigation.selectRow().invoke("View");
+        Application.waitUntilVisible();
+
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/runtime/TransactionsMetricsTestCase.java]: Please move this at the end of `before()` method.

File: basic/src/test/java/org/jboss/hal/testsuite/test/runtime/TransactionsMetricsTestCase.java [hunk 6]
```diff
@@ -59,6 +87,9 @@ public void successRationMetrics() {
 
     @Test
     public void failureOriginMetrics() {
+        navigation.selectRow().invoke("View");
+        Application.waitUntilVisible();
+
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/runtime/TransactionsMetricsTestCase.java]: Please move this at the end of `before()` method.

**Comment by cypo721** [comment]
Moved.

**Pull request merged**